kernelci.org/content/en/labs/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Labs"
+date: 2024-07-03
+description: "Connecting labs to KernelCI"
+---
+
+Anyone can connect their labs to KernelCI. Once a Lab is connected to KernelCI, it can execute KernelCI tests.
+
+Whether the Lab will receive specific test configurations or just a general guideline of what tests to run is depended on each Labs preference. If the lab is a LAVA lab, for example, the KernelCI [LAVA runtime](../api_pipeline/pipeline/connecting-lab) will generate test jobs for each single test that is configured to run. On the other hand, if labs are implementing their own runtime (or bridge to Maestro APIs) they may choose to generate the individual test jobs themselves.
+
+If are interesting it just sending the results of test runs in your infrastructure, you should look at sending the data to [KCIDB](../kcidb) - KernelCI common database for results.
+
+There are a few ways labs can be connected to KernelCI. The documentation for connecting labs in the new architecture is evolving as we add labs.
+
+* [Connecting a LAVA lab](../api_pipeline/pipeline/connecting-lab)
+* Writing your own bride/runtime using Maestro APIs (TBD)
```

Comment:
It could be useful to link here @JenySadadia's [examples of bridging with Maestro](https://github.com/kernelci/kernelci-project/issues/349#issuecomment-2097641023)

Reply:
Added a link to that GitHub Issue.